docs/en/reference/information_schema/tables_config.md

```diff
@@ -0,0 +1,24 @@
+---
+displayed_sidebar: "English"
+---
+
+# tables_config
+
+`tables_config` provides information about the configuration of tables.
+
+The following fields are provided in `tables_config`:
+
+| **Field**        | **Description**                                              |
+| ---------------- | ------------------------------------------------------------ |
+| TABLE_SCHEMA     | Name of the database that stores the table.                  |
+| TABLE_NAME       | Name of the table.                                           |
+| TABLE_ENGINE     | Engine type of the table.                                    |
+| TABLE_MODEL      | Data model of the table. Valid values: "DUP_KEYS", "AGG_KEYS", "UNQ_KEYS" or "PRI_KEYS". |
+| PRIMARY_KEY      | The primary key of a Primary Key table or a Unique Key table. An empty string is returned if the table is not a Primary Key table or a Unique Key table. |
+| PARTITION_KEY    | The partitioning columns of the table.                       |
+| DISTRIBUTE_KEY   | The bucketing columns of the table.                          |
+| DISTRIBUTE_TYPE  | The data distribution method of the table.                   |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/83c82d60-2ee2-4961-8b99-b71906070f6f)
Remove these "The", because the other cells in this table all start without the article "The".